src/ui/Pager/Pager.ts

```diff
@@ -432,6 +438,7 @@ export class Pager extends Component {
       .withElement(nextButton)
       .withLabel(l('Next'))
       .withSelectAction(() => this.handleClickNext())
+      .withRole('listitem')
```

Comment:
The `listitem` role doesn't imply that this element is selectable, and the `span` a child doesn't either. We should have a selectable role somewhere.

also, IMO, the error reported by the automated tool seems pretty irrelevant.